Add GET_ATTR instruction

iseeyuan opened the pull request:
The prim::GetAttr and prim::SetAttr operators depend on node. However, in lite interpreter there will be no node dependency. Promote them to a first-class instruction. 
Tests:
GET_ATTR is covered by testModuleDefine()
SET_ATTR is covered by testScriptObject()

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #25187 [Lite Interpreter] Bytecode export flow
* **#25151 Add GET_ATTR instruction**
* #25148 Pull instruction definitions out of interpreter.cpp.

Differential Revision: [D17076412](https://our.internmc.facebook.com/intern/diff/D17076412)

Commit:
Add GET_ATTR instruction

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr operator depends on node. However, in lite interpreter there will be no node dependency. Promote the operator to a first-class instruction.

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr operator depends on node. However, in lite interpreter there will be no node dependency. Promote the operator to a first-class instruction.

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr operator depends on node. However, in lite interpreter there will be no node dependency. Promote the operator to a first-class instruction.

zdevito reviewed (commented):
Looks good, but we need to do some additional changes for consistency:
* Implement SetAttr as an instruction as well, it is weird to have one be a primitive and one be an instruction
* Delete prim::{Set,Get}Attr from register_prim_ops, since they are no longer used

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr operator depends on node. However, in lite interpreter there will be no node dependency. Promote the operator to a first-class instruction.

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr operator depends on node. However, in lite interpreter there will be no node dependency. Promote the operator to a first-class instruction.

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr and prim::SetAttr operators depend on node. However, in lite interpreter there will be no node dependency. Promote them to a first-class instruction. 
Tests:
GET_ATTR is covered by testModuleDefine()
SET_ATTR is covered by testScriptObject()


Differential Revision: [D17076412](https://our.internmc.facebook.com/intern/diff/D17076412)

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr and prim::SetAttr operators depend on node. However, in lite interpreter there will be no node dependency. Promote them to a first-class instruction. 
Tests:
GET_ATTR is covered by testModuleDefine()
SET_ATTR is covered by testScriptObject()


Differential Revision: [D17076412](https://our.internmc.facebook.com/intern/diff/D17076412)

Commit:
Update on "Add GET_ATTR instruction"

The prim::GetAttr and prim::SetAttr operators depend on node. However, in lite interpreter there will be no node dependency. Promote them to a first-class instruction. 
Tests:
GET_ATTR is covered by testModuleDefine()
SET_ATTR is covered by testScriptObject()


Differential Revision: [D17076412](https://our.internmc.facebook.com/intern/diff/D17076412)
